### ISSUE TITLE
Question regarding Dreambooth config

### ISSUE THREAD

**Issue by SlavyanDesu** [Linaqruf/kohya-trainer#135]
Is `instance_token` same as `instance_prompt`? If so, let's say I want to create LoRA for [Misono Mika](https://bluearchive.fandom.com/wiki/Misono_Mika) and the activator tag is `mika (blue archive)`. It's the same tag as Booru does. Do I still need underscore as a space or just keep it? Thanks in advance.

**Comment by SlavyanDesu** [comment]
Thanks for the reply @noriwaru.
Yes, I already put the prompt/tag on Dreambooth Config, so do the caption file for each image.

For your info, here are my token settings:
```
instance_token="mika"
class_token="1girl"
```
Here are my example of `1.txt`:
`1girl, mika, halo, solo, wings, smile, hair bun`

**Comment by noriwaru** [comment]
The original sd-script names the learning folder "number of iterations, instance_token and class_token" (e.g. 10_john girl). However, if there is a text file in the folder, the folder name is ignored and the tags entered in the text are used for training.
kohya-trainer  Tested.
5.2 Dataset Config Dreambooth Config
train_repeats:10
reg_repeats:10
instance_token: car
class_token: dog
Text tags - train image= cpc, reg image= scs
train image is circular, regularized image is rectangular

The generated results were drawn as a circle in cpc and a rectangle in scs.
In other words, when using text files, I think that instance_token: car
 instance_token: dog is ignored when using text files.

**Comment by SlavyanDesu** [comment]
I see, so that means it's OK to leave `instance_token` and `class_token` empty and stick to `.txt` files?

**Comment by SlavyanDesu** [comment]
Thanks for your answer! @noriwaru
